packages/test/src/integration-tests/test-update-interceptors.ts

```diff
@@ -0,0 +1,48 @@
+import { Next, UpdateInput, WorkflowInboundCallsInterceptor } from '@temporalio/workflow';
+import { helpers, makeTestFunction } from './helpers';
+import { update, workflowWithUpdates, workflowsPath } from './workflows';
```

Comment:
The nice thing about this type of test is that you can inline the workflows with the test files. Importing the workflow beats the point IMHO.